Assets/Scripts/UI/PauseMenu/PauseMenu.cs

```diff
@@ -5,7 +5,7 @@
 
 public class PauseMenu : MonoBehaviour
 {
-    private static bool GameIsPaused = false;
+    public static bool GameIsPaused = false;
```

Comment:
Please do not make this variable public and use getters/setters to access/modify the value of the variable